src/process/native/win32.js

```diff
@@ -1,8 +1,88 @@
-import { exec } from 'child_process';
-
-export const getProcesses = () => new Promise(res => exec(`wmic process get ProcessID,ExecutablePath /format:csv`, (e, out) => {
-  res(out.toString().split('\r\n').slice(2).map(x => {
-    const parsed = x.trim().split(',').slice(1).reverse();
-    return [ parseInt(parsed[0]) || parsed[0], parsed[1] ];
-  }).filter(x => x[1]));
-}));
+import koffi from 'koffi';
+
+// Load Windows API
+const psapi = koffi.load('psapi.dll');
+const kernel32 = koffi.load('kernel32.dll');
+const ntdll = koffi.load('ntdll.dll');
+
+// Define Alias
+const DWORD = koffi.alias('DWORD', 'uint32_t');
+const BOOL = koffi.alias('BOOL', 'int32_t');
+const HANDLE = koffi.pointer('HANDLE', koffi.opaque())
+
+const UNICODE_STRING = koffi.struct('UNICODE_STRING', {
+  Length: 'uint16',
+  MaximumLength: 'uint16',
+  Buffer: HANDLE
+});
+
```

Comment:
```suggestion

// https://www.geoffchappell.com/studies/windows/km/ntoskrnl/api/ex/sysinfo/process_id.htm
```
It's officially undocumented but you could put this here, it has some good information